Added ErodeBlend brush

Aime23 opened the pull request:
Implementation is a mix between ShoukaSeikyo's BetterBrushes implementation (my interpretation on how it works) and [FastAsyncVoxelSniper](https://github.com/IntellectualSites/FastAsyncVoxelSniper/blob/master/src/main/java/com/thevoxelbox/voxelsniper/brush/ErodeBlendBrush.java) implementation

Resolve #7, resolve #15

Aime23 committed:
Added ErodeBlend brush

A mix between ShoukaSeikyo's BetterBrushes implementation and FastAsyncVoxelSniper implementation.

ShoukaSeikyo commented:
I can concur your implementation is correct. (It is basically the same I did at the time.)

> blendBallBrush.excludeAir = false;
I would only remove this line as it is redundant; `blendBallBrush.arrow(v)` already sets `excludeAir` to `false`.

Aime23 committed:
Added arguments registration and custom info messages

Added a getter for the erode brush's preset name for the messages.
Removed useless excludeAir due to redundancy

Aime23 commented:
The info messages are a mix between BlendBall and Erode ones, and they looks like that :
/b eb
![Info](https://user-images.githubusercontent.com/30575474/84131255-9b67f180-aa44-11ea-8bc5-01f083a096a7.png)
/b eb info
![b eb info](https://user-images.githubusercontent.com/30575474/84131312-ae7ac180-aa44-11ea-9f8b-6db025addae8.PNG)

Aime23 committed:
Removed an error when parsing parameters
